crates/sui-graphql-client/src/lib.rs

```diff
@@ -672,6 +672,41 @@ impl Client {
             Ok(None)
         }
     }
+
+    /// Execute a transaction.
+    ///
+    /// The `signatures` should be a list of signatures for the transaction Base64 encoded.
+    /// The `tx_bytes` should be the transaction bytes Base64 encoded.
+    pub async fn execute_tx(
+        &self,
+        signatures: Vec<&str>,
+        tx_bytes: &str,
```

Comment:
I think the better way to handle this API would be to pass in the concrete type and have the function convert to the base64 encoded string

Reply:
Fixed